stonesoup/models/driver.py

```diff
@@ -0,0 +1,251 @@
+from typing import Callable, Generator, Optional, Union
```

Comment:
This file (stonesoup.models.driver) needs to be added to the docs/source/stonesoup.models.rst file for it to appear in the documentation.